base_report_to_printer/tests/test_ir_actions_report.py

```diff
@@ -73,6 +73,7 @@ def test_print_action_for_report_name_returns_if_report(self):
         """ It should return correct serializable result for behaviour """
         with mock.patch.object(self.Model, '_get_report_from_name') as mk:
             res = self.Model.print_action_for_report_name('test')
+            del res['id']  # HELP!
```

Comment:
What if you add below to `expect` dict: `'id': behaviour.id` instead?

Reply:
I tried it, but I don't think it will work, behaviour is just a method, not an instance of a report.

Reply:
actually, now that I think about it, probably the way I did the test is best for now.  print_action_for_report_name adds id and is the only function that adds id becuase it is only needed for the rpc query.  By deleting that key, we are confirming that it is adding that key, and only that key.

Reply:
@Yajo I fixed it so its much better